src/app/projects/components/project-settings/project-settings.component.ts

```diff
@@ -173,6 +184,7 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy, AfterViewIni
         }
       })
     }
+    this.isManaged = ConfigManager.getIsManaged();
```

Comment:
i think we need to wait for the configmanager to be initialized first ngOnInit is usually to early